utils/shipping-actions.js

```diff
@@ -154,3 +160,43 @@ export function selectOtherPickup() {
     .click()
   cy.get('.pkpmodal-details-confirm-btn').click()
 }
+
+export function fillShippingPreviewPickupAddress(account) {
+  cy.get('#find-pickup-link').click()
+
+  if (
+    [
+      ACCOUNT_NAMES.CLEAN_NO_MAPS,
+      ACCOUNT_NAMES.NO_LEAN,
+      ACCOUNT_NAMES.INVOICE,
+    ].includes(account)
+  ) {
+    cy.waitAndGet('#pkpmodal-search #ship-postalCode', 3000).type('22071060')
+  } else {
+    cy.waitAndGet('#pkpmodal-search input', 3000).type('Praia de Botafogo, 300')
+
+    cy.get('.pac-item')
+      .first()
+      .trigger('mouseover', { force: true })
+
+    cy.get('.pac-item')
+      .first()
+      .click({ force: true })
+  }
+  cy.get('.pkpmodal-points-list .pkpmodal-pickup-point-main').click()
+
+  cy.get('.pkpmodal-details-confirm-btn').click()
```

Comment:
Do not copy-paste code. Remember why we're doing this refactor in the first place.
Extract this to a function and use it in both here and in `fillPickupAddress`.

Reply:
You're right! I'll do that.